framework/src/modules/chain/migrations/sql/20190705101215_create_block_temp_table.sql

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+
+/*
+  DESCRIPTION: Creates meta table for BFT.
+
+  PARAMETERS: None
+*/
+
+CREATE TABLE IF NOT EXISTS "block_temp" (
+	"id" character varying(20) NOT NULL,
```

Comment:
```suggestion
	"id" VARCHAR(20) PRIMARY KEY,
```

Reply:
Regarding Nazar's comments, the table name in the SQL files should be changes as well to `temp_block`